docs/tutorials/cargo_tests/baked/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+# This file is part of ICU4X. For terms of use, please see the file
+# called LICENSE at the top level of the ICU4X source tree
+# (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+##########################################
+# PLEASE KEEP THIS IN SYNC WITH cargo.md #
+##########################################
+
+[package]
+name = "demo_baked"
```

Comment:
Naming. Currently there's
* "cargo tutorial" in text, make jobs
* "cargo" in CI jobs, make jobs
* "cargo_tests" in the directory name
* "demo_foo" in the crate names

Can we align everything to tutorial?
* `ci-job-foo-cargo-tutorial-bar`
* `docs/tutorials/cargo`
* crate name `baked_tutorial`/`tutorial_baked`
  * Or just `baked`, as it's in the tutorial workspace?

Reply:
did `tutorial_*`

Reply:
What about the `cargo_tests` directory name?

Reply:
Not sure about the directory name. `cargo_tests` is probably not right, but I don't like just `cargo` because then the directory looks too similar to `.cargo` which has real meaning. I could do

- `crates`
- `cargotoml`
- `cargo_examples`
- `examples`

Reply:
I like `crates` or `cargo_tutorials`, although that's a bit repetitive. I'm happy with `examples` if you replace all the `-tutorials` with `-examples` (which I don't expect you to) 🙃